docs/topics/Threads.md

```diff
@@ -163,24 +163,42 @@ Additionally, when a client loses access to a channel they are not removed from
-- Threads can be created in a forum channel. All threads in a forum are of type `PUBLIC_THREAD`. These threads and messages within the thread have the same gateway events as threads in a normal text channel.
-- The APIs for loading active & archived threads, joining & leaving a thread, and loading who is in a thread are unchanged and work the same for threads in a forum.
-- The API to create a thread in a forum will create _both_ a thread and message in the same call, and as such requires passing in parameters for both the thread and message. The name and behavior of parameters is the same as they are for the existing create thread/message endpoints to simplify integrating with it.
-- The message created by that API call will have the same id as the thread.
-- The `last_message_id` field on the forum channel object tracks the id of the most recently created thread. It has the same behavior and requirements as it does for messages, namely that you will not receive a `CHANNEL_UPDATE` when it is changed. Instead clients should update the value when receiving [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create).
-- The `message_count` and `total_message_sent` fields on threads created in a forum will increment on `MESSAGE_CREATE` and will decrement on `MESSAGE_DELETE`/`MESSAGE_DELETE_BULK`. There will be no `CHANNEL_UPDATE` event through gateway notifying those fields' changes (similar to `last_message_id` changes). Clients should update those values when receiving corresponding events.
-- The `topic` field on a forum channel is what is shown in the "Guidelines" section visually
-- The `rate_limit_per_user` field currently behaves the same as in a text channel, limiting how frequently threads can be created. There is a new `default_thread_rate_limit_per_user` field that can be set on the forum as well, which will limit how often messages can be sent _in a thread_. This field is copied into `rate_limit_per_user` on the thread at creation time.
-- Threads in a forum have the same permissions behavior as threads in a text channel, inheriting all permissions from the parent channel, with just one exception: Creating a thread in a forum channel only requires the permission that is currently named `SEND_MESSAGES`.
-- The first message in a forum thread can contain additional markdown for bulleted list and headings.
-- A thread can be pinned within a forum. A thread that is pinned will have the `(1 << 1)` flag set. Archiving a pinned thread will unset the flag. A pinned thread will not auto archive.
-- Forums can specify `available_tags` that can be set on individual threads via the `applied_tags` field.
+### Creating Threads in Forum Channels
+
+Threads appear as forum posts within forum channels, and are created the same way as threads in text channels. However, private threads are currently not supported in forum channels so all threads created must be of type `PUBLIC_THREAD`.
```

Comment:
> and are created the same way as threads in text channels

There are some small differences actually, for example when creating a thread in a forum you *must* also create a message at the same time in that API call, so there's some extra args required.

> private threads are currently not supported in forum channels so all threads created must be of type `PUBLIC_THREAD`

I'd suggest tweaking this a tad because it implies that we might support private threads in the future and that you have pass in the type. We actually don't even expose a `type` parameter for the create-forum-thread endpoint, we just force it to be a public thread in the API code